packages/obojobo-document-engine/server/question-bank.js

```diff
@@ -0,0 +1,125 @@
+const DraftNode = oboRequire('models/draft_node')
+const _ = require('underscore')
+const logger = oboRequire('logger')
+const { getRandom } = require('./util')
+
+const SELECT_SEQUENTIAL = 'sequential'
+const SELECT_RANDOM = 'random'
+const SELECT_RANDOM_UNSEEN = 'random_unseen'
+
+const CHOOSE_ALL = 'all'
+
+class QuestionBank extends DraftNode {
+	constructor(draftTree, node, initFn) {
+		super(draftTree, node, initFn)
+	}
+
+	buildAssessment(questionUsesMap) {
+		const { choose, select } = this.getContentValues()
```

Comment:
Could possibly be set up in constructor so we could use this.choose/ this.select